ApiClient/Tests/ParsingTests.swift

```diff
@@ -52,8 +52,7 @@ class ApiClientTests: XCTestCase {
         
         let decoder = JSONDecoder()
         do {
-            let pageItem = try decoder.decode(JSONPageItem.self, from: data)
-            print(pageItem)
+            _ = try decoder.decode(JSONPageItem.self, from: data)
```

Comment:
How are these changes related to the String to URL change?

Reply:
As well as improving test coverage I cleaned up the code in the tests.